framework/bee-py/bee_agent/tools/mcp_tools.py

```diff
@@ -0,0 +1,181 @@
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
General point - how do we keep framework in sync (and ensure issues etc are fixed) in future. Will we address this as part of our overall repo refactoring? If we want to depend on select features in another repo we may want a mechanism to support this (subproject, review process, tagging etc).